probatus/feature_elimination/feature_elimination.py

```diff
@@ -98,10 +98,14 @@ def __init__(self, clf, step=1, min_features_to_select=1, cv=None, scoring='roc_
                 iteration. It is recommended to use float, since it is faster for a large number of features, and slows
                 down and becomes more precise towards less features. Note: the last round may remove fewer features in
                 order to reach min_features_to_select.
+                If columns_to_keep parameter is specified in the fit method, step is the number of features to remove after 
```

Comment:
Maybe this sentence is not necessary, it seems logical that these features are taken into consideration.

Reply:
I think, it is better if we are explicit. What do you think ?

Reply:
Let's be explicit then and make the change that you proposed